sources/packages/web/src/components/layouts/aest/AESTApplicationSideBar.vue

```diff
@@ -99,6 +99,7 @@ export default defineComponent({
     const { getISODateHourMinuteString } = useFormatters();
     const relatedParentPartners = ref([] as MenuItemModel[]);
     const applicationHistory = ref([] as MenuItemModel[]);
+    const currentApplicationId = ref<number>();
```

Comment:
There is no need to have it declared as reactive property and to be at this scope. Please change it to a regular property and use it only inside the `onMounted` scope.